README.md

```diff
@@ -92,51 +96,74 @@ provider "volterra" {
 }
 
 module "app-delivery-network" {
-  source             = "volterraedge/app-delivery-network/volterra"
-  adn_name           = var.name
-  volterra_namespace = local.namespace
-  app_domain         = var.app_fqdn
+  source               = "volterraedge/app-delivery-network/volterra"
+  adn_name             = var.name
+  volterra_namespace   = local.namespace
+  app_domain           = var.app_fqdn
+  disable_js_challenge = var.disable_js_challenge
 }
 
 output "adn_app_url" {
   value = module.app-delivery-network.app_url
 }
 ```
 ---
-## Requirements
+## How to update Requirements:
 
+Delete everything below the three dashes `---` in this Readme file and then run `terraform-docs md tbl . >> README.md` to update to the latest requirements
```

Comment:
@cohenaj194: Can we not add this in this readme, this will directly show up here https://registry.terraform.io/modules/volterraedge/app-delivery-network/volterra/latest

May be we can create a new file with developer.md file where we can have these notes

Reply:
This has been removed and moved to ` developer.md `